terror_error.go

```diff
@@ -31,6 +31,45 @@ type ErrCodeText string
 type ErrorID string
 type RFCErrorCode string
 
+var class2RFCCode = map[int]string{
```

Comment:
Please add a reference link to these number and string.

Reply:
fixed, PTAL @overvenus

Reply:
It seems to be a bad idea to introduce tidb only classes as it is a commonly used library...
I think it would be better to add a register function to allow TiDB to set it outside.

Reply:
OK, we can make further improvements.